pkg/cluster/cluster.go

```diff
@@ -20,7 +20,7 @@ type Cluster struct {
 	config           *config.Control
 	runtime          *config.ControlRuntime
 	managedDB        managed.Driver
-	etcdConfig       endpoint.ETCDConfig
+	EtcdConfig       endpoint.ETCDConfig
```

Comment:
I'm a little sketched out by exporting this; none of the other fields are exported and the fact that we're now exposing things off it me suspect we're poking at things across modules that we shouldn't. Can you comment as to why we need this exposed?

Reply:
We needed to expose this in order for the server to call Save correctly later. Due to circular import issues, it was impossible to simple save the cluster pointer as a whole. Now the server stores only the EtcdConfig and gives it exclusively to the Save function.

Reply:
got it!